src/watertap_contrib/seto/unit_models/surrogate/vagmd_surrogate.py

```diff
@@ -0,0 +1,1233 @@
+        tmp_dict["defined_state"] = True
+
+        self.condenser_in_props = (
+            self.config.property_package_seawater.state_block_class(
+                self.flowsheet().config.time,
+                doc="Material properties of condenser inlet water",
+                **tmp_dict,
+            )
+        )
+
+        """
+        Add block for water at the condenser outlet
+        """
+        tmp_dict["defined_state"] = False
+
+        self.condenser_out_props = (
+            self.config.property_package_seawater.state_block_class(
+                self.flowsheet().config.time,
+                doc="Material properties of condenser outlet water",
+                **tmp_dict,
+            )
+        )
+
+        """
+        Add block for the inlet cooling water 
+        """
+        self.cooling_in_props = self.config.property_package_seawater.state_block_class(
+            self.flowsheet().config.time,
+            doc="Material properties of the inlet cooling water",
+            **tmp_dict,
+        )
+
+        # Specify the concentration and temperature with any value for initialization,
+        # because only volumetric flow rate is known and the temperature will be released and calculated afterwards
+        self.cooling_in_props.calculate_state(
+            var_args={
+                # Cooling water volumetric flow rate, fixed to 1265L/h, to maintain vacuum pressure inside the MD module around 200 mbar
+                ("flow_vol_phase", "Liq"): pyunits.convert(
+                    1265 * pyunits.L / pyunits.h, to_units=pyunits.m**3 / pyunits.s
+                ),
+                ("conc_mass_phase_comp", ("Liq", "TDS")): 50,
+                ("temperature", None): 25 + 273.15,
+                ("pressure", None): 101325,
+            },
+            hold_state=True,
+        )
+        self.cooling_in_props[0].temperature.unfix()
+
+        """
+        Add block for the outlet cooling water
+        """
+        self.cooling_out_props = (
+            self.config.property_package_seawater.state_block_class(
+                self.flowsheet().config.time,
+                doc="Material properties of the outlet cooling water",
+                **tmp_dict,
+            )
+        )
+        # Specify the concentration and temperature with any value for initialization,
+        # because only volumetric flow rate is known and the temperature will be released and calculated afterwards
+        self.cooling_out_props.calculate_state(
+            var_args={
+                # Cooling water volumetric flow rate, fixed to 1265L/h, to maintain vacuum pressure inside the MD module around 200 mbar
+                ("flow_vol_phase", "Liq"): pyunits.convert(
+                    1265 * pyunits.L / pyunits.h, to_units=pyunits.m**3 / pyunits.s
+                ),
+                ("conc_mass_phase_comp", ("Liq", "TDS")): 50,
+                ("temperature", None): 25 + 273.15,
+                ("pressure", None): 101325,
```

Comment:
These should be Params. Also, what if we wanted to scale up and, say, assume multiple modules in parallel? Wouldn't flowrate change?

Reply:
Similar to my response above that this unit model is for 1 module only. I've considered to make it suitable for scaling up but I gave it up for the following reasons:
1. The surrogate equations use the properties of water within one module. If allowing scaling up, the state blocks will need to be duplicated for the actual flows, which is fine but could looks a bit chunky. 
2. My initial thought on the MD applications would be batch/semi-batch operation (the mode that was delivered for SETO project), because once-through process has very low recovery ratio. So I didn't expect the unit model to be used alone, but to be integrated in a multi-period structure, which considers the capacity and cost in the multi-period level. 

Of course someone might want to use it along for some reason, in that case, maybe we can have another unit model for that, and this model will be like "reverse_osmosis_base" that are used in 'oaro' or 'ro_0D'.